src/activity-logger/background/log-page-visit.test.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-env jest */
+/* eslint import/namespace: "off" */
+
+import maybeLogPageVisit from 'src/activity-logger/background/log-page-visit'
+import * as storePage from 'src/page-storage/store-page'
+import db from 'src/pouchdb'
+
+afterAll(async () => {
+    await db.destroy()
+})
+
+describe('maybeLogPageVisit', () => {
```

Comment:
I would definitely like to test functions that are not exported. The question is how to do this.
- One option is something like [rewire](https://github.com/jhnns/rewire), but it should work with our babel setup, which rewire does not support.
- Another option is to export everything (that we wish to test) from each file, but then have extra files that import and re-export only the ones that are really meant to be exported. Most logically the public API of files in a folder would be exposed in `index.js`, which could perhaps pull and export the default exports from files around it. It's just one logical structure that pops up in my mind now.
- A third option is to just export things you want to test and not bother about the difference of public/private functions for now. I suggest you take this approach now, and we move to a neater approach later (probably as part of a larger organisational restructuring, see #112).